tests/Schedule/CancelDayRuleTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace GroupLife\Core\tests;
+
+use GroupLife\Core\Schedule\CancelDayRule;
+use PHPUnit\Framework\TestCase;
+
+class CancelDayRuleTest extends TestCase
+{
+
+    public function testIncludedDays()
+    {
+        $dateFrom = new \DateTime('2021-01-01');
+        $oneMonth = new \DateInterval('P1M');
+        $cancelDays = new CancelDayRule(['2021-01-01' => '10:00', '2021-01-02' => '09:00']);
```

Comment:
Now, when I can see the `CancelDayRule` interface, I propose to simplify this.

Let's `CancelDayRule` contain exactly one day rule. When there are several cancelled days we'll include them as several "cancel day rules".


```
$cancelledHoliday = new CancelDayRule('2021-12-12', '10:00');
```